IgOmeProfiling_pipeline.py

```diff
@@ -42,7 +42,7 @@ def run_pipeline(fastq_path, barcode2samplename_path, samplename2biologicalcondi
 
         module_parameters = [fastq_path, first_phase_output_path, first_phase_logs_path,
                              barcode2samplename_path, left_construct, right_construct,
-                             max_mismatches_allowed, min_sequencing_quality, first_phase_done_path,
+                             max_mismatches_allowed, min_sequencing_quality, minimal_length_required,first_phase_done_path,
```

Comment:
Mismatch with the definition in read_filtration/module_wrapper.py.
You define that as a named parameter (starts with --) and pass it here as positional parameter.
Also the order is incorrect/doesn't match, you are passing minimal length as done path.

To summarize, this change is wrong and doesn't work